package/PartSeg/common_backend/base_settings.py

```diff
@@ -202,6 +225,28 @@ def __init__(self):
         self.label_color_dict = LabelColorDict(self.get_from_profile("custom_label_colors", {}))
         self.cached_labels: Optional[Tuple[str, np.ndarray]] = None
 
+    @property
+    def theme_name(self) -> str:
+        return self.get_from_profile("theme", "light")
+
+    @property
+    def style_sheet(self):
+        palette = napari.utils.theme.palettes[self.theme_name]
+        palette["canvas"] = "black"
+        return napari_template(get_stylesheet(), **palette)
+
+    @theme_name.setter
+    def theme_name(self, value: str):
+        if value not in napari.utils.theme.palettes:
+            raise ValueError(f"Unsupported theme {value}. Supported one: {self.theme_list()}")
+        if value == self.theme_name:
+            return
+        self.set_in_profile("theme", value)
+        self.theme_changed.emit()
+
+    def theme_list(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=5714517)